modules/ROOT/pages/zero-to-hero.adoc

```diff
@@ -0,0 +1,293 @@
+= Zero to Hero
+
+The Enterprise Contract can be used to verify an image that has already been signed and attested by
+a build system. This document assumes you do not have such an image. It walks you through the
+process of signing and attestating an image, then how to use EC to validate those operations.
+
+== Required Tooling
+
+Before starting, the following tooling is required:
+
+* https://github.com/sigstore/cosign#installation[cosign]
+* https://enterprisecontract.dev/docs/user-guide/main/cli.html[ec cli]
+* https://github.com/containers/skopeo[skopeo] or
+  https://github.com/google/go-containerregistry/tree/main/cmd/crane[crane]
+* https://docs.docker.com/engine/reference/run/[docker] or https://podman.io/[podman] (optional)
+
+== Container Repository
+
+During this guide, image signatures and attestations will be pushed to the container repository
+where the image resides. Using a local registry is ideal for testing purposes. If you have a
+container runtime installed, e.g. https://docs.docker.com/engine/reference/run/[docker] or
+https://podman.io/[podman], you can start one with ease:
+
+[.console-input]
+[source, bash]
+----
+docker run -it -d -p 5000:5000 --restart=always --name registry registry:2
+# or with podman:
+# podman run -it -d -p 5000:5000 --restart=always --name registry registry:2
```

Comment:
Consider having both commands listed separately. That way the user can copy/paste which one they would like to use.

Reply:
I had it like that originally, but then I realized that folks usually just copy and paste things when it comes to guides like this. I wanted to avoid someone accidentally running both commands.

Reply:
Makes sense.